host-interaction/network/connectivity/check-internet-connectivity-via-wininet.yml

```diff
@@ -11,12 +11,14 @@ rule:
     examples:
       - 648FC498110B11B4313A47A776E6BA40:0x6633F0
   features:
-    - and:
-      - or:
-        - api: wininet.InternetGetConnectedState
-        - api: wininet.InternetCheckConnection
-      - optional:
-        - mnemonic: cmp
+    - or:
+      - and:
         - or:
-          - number: 0
-          - number: 1
+          - api: wininet.InternetGetConnectedState
+          - api: wininet.InternetCheckConnection
+        - optional:
+          - mnemonic: cmp
+          - or:
+            - number: 0
+            - number: 1
```

Comment:
do you know what these constants represent?

Reply:
yes I believe these represent `TRUE` and `FALSE` - I'll get these added.